modules/engage-theodul-plugin-tab-downloads/pom.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The `maven-dependency-plugin` complains about several dependencies in here.
You are also not really using any Jasmine or JUnit tests, are you?